2.7/s2i/bin/run

```diff
@@ -35,6 +35,13 @@ function get_default_web_concurrency() {
   echo $default
 }
 
+APP_HOME=$(readlink -f ${APP_HOME:-.})
```

Comment:
Double quotes needed around APP_HOME

Reply:
When trying this APP_HOME=$(readlink -f ${"APP_HOME":-.}) it ends up with an error bash: ${"APP_HOME":-.}: bad substitution, maybe I don't understand your point here correctly

Reply:
I guess `APP_HOME=$(readlink -f "${APP_HOME:-.}")`